src/hooks/custom-hooks/usePoiPoaStatus.ts

```diff
@@ -12,7 +12,9 @@ const usePoiPoaStatus = () => {
         const documentStatus = data?.authentication?.document?.status;
         const identityStatus = data?.authentication?.identity?.status;
         const isP2PPoaRequired = data?.p2p_poa_required;
-        const isPoaAuthenticatedWithIdv = data?.status.includes('poa_authenticated_with_idv');
+        const isPoaAuthenticatedWithIdv =
+            data?.status.includes('poa_authenticated_with_idv') ||
+            data?.status.includes('poa_authenticated_with_idv_photo');
```

Comment:
Aren't both of them authenticated using IDV?

Reply:
BE has renamed the flag to `poa_authenticated_with_idv_photo` but we are keeping the old one to make it backward compatible. As soon as the BE card is released we will remove`poa_authenticated_with_idv`.